Assets/Scripts/Evasion/OrthogonalEvasion.cs

```diff
@@ -0,0 +1,86 @@
+using UnityEngine;
+
+// The agent will try to evade its pursuer by turning its velocity vector to be orthogonal to the
+// pursuer's velocity vector.
+//
+// Turning orthogonal to the pursuer means that the pursuer needs to apply a normal acceleration to
+// intercept the agent, incurring the maximum possible lift-induced drag.
+public class OrthogonalEvasion : EvasionBase {
+  public OrthogonalEvasion(IAgent agent) : base(agent) {}
+
+  // Determine whether to perform any evasive maneuvers.
+  public override bool ShouldEvade(IAgent pursuer) {
+    // Default evasion range threshold in meters.
+    const float defaultEvasionRangeThreshold = 1000f;
+
+    if (!(Agent.AgentConfig?.DynamicConfig?.FlightConfig?.EvasionConfig?.Enabled ?? false)) {
+      return false;
+    }
+
+    float evasionRangeThreshold =
+        Agent.AgentConfig?.DynamicConfig?.FlightConfig?.EvasionConfig?.RangeThreshold ??
```

Comment:
On line 16 we already guarded the null EvasionConfig, so there is no need for this redundant null check.